packages/core/src/ceramic.ts

```diff
@@ -27,11 +34,11 @@ import { PathTrie, TrieNode, promiseTimeout } from './utils'
 
 import EthereumAnchorService from "./anchor/ethereum/ethereum-anchor-service"
 import InMemoryAnchorService from "./anchor/memory/in-memory-anchor-service"
-import { PinningBackendStatic } from "@ceramicnetwork/common"
-
 
 import { randomUint32 } from '@stablelib/random'
 
+const DEFAULT_BASE_DOC_CACHE_LIMIT = 100; // number of base docs stored in the cache
```

Comment:
This seems extremely small. The nodes on our infra needs to keep track of documents for all of our users > 15K, and each user will have multiple documents.

Reply:
I'll increase it to `500`. I added an option for setting a cache limit as well